tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -81,7 +81,12 @@ def self.metadata
               type: "filename",
               required: false,
               default: "output/qualys_was",
-              description: "If set, will write a file upon completion. Path is relative to #{$basedir}" }
+              description: "If set, will write a file upon completion. Path is relative to #{$basedir}" },
+            { name: "match_ScannerIdentifier_with_vuln_def_name",
+              type: "boolean",
+              required: false,
+              default: false,
+              description: "If true, the Scanner Description in UI will be coming from the vuln_defs' name in original JSON file matched with scanner_identifier. Each imported finding unless have the exact same qid, id, and name from source data, will be created as new one." }
```

Comment:
This looks like it could be confusing for customers as it's describing a backend process.
I'd suggest something like this but interested in other people's feedback:

"If true, the finding scanner_identifier and the vuln_def name will match the format 'QID - ID - Name'. If a connector first ran without this parameter and it is enabled in a later run, it will result in automatic resolution of the existing findings with the previous format ('QID - ID') and creation of new findings with the new format within CVM."

Reply:
Thank you ! Your concern makes sense.

Reply:
I think the UI explanation is not needed since the data would be coming from the same place (SVD description/solution) and this is only impacting SVD selection process.